HACKING.md

```diff
@@ -1,288 +1,167 @@
-# Goldstone Hacking Guide
+# Goldstone Server Hacking Guide
 
-This explains how to install and run Goldstone locally, so you can do code development on it.
 
-The recommended steps are:
+This document explains how to install and run Goldstone Server locally (mostly in docker containers), so you can do code development on the project.  The instructions assume a Mac OS X Yosemite development environment with [homebrew](http://brew.sh/) and [Virtualbox](https://www.virtualbox.org/wiki/Downloads) installed.
 
-1. Fork this repository on GitHub.
-2. Follow the "Installation" instructions to set up your development environment.
-3. Verify that you can build a working Goldstone application before you make any edits!
-4. Make your improvements in your fork. Remember to include unit tests, and use tox (with the supplied `tox.ini`) to verify that the codebase still passes pep8, pylint, and all unit tests.
-5. Submit a pull request to us. Be sure to include a problem description and an overview of your solution.
+[TOC]
 
-Remember that an instance of Goldstone running locally can be used to monitor a
-local *or remote* OpenStack installation. The server on which Goldstone runs is
-independent from the OpenStack cloud's location.
+## Prerequisites
 
+Install various prerequisite packages:
 
-## Installation
+    $ brew update
+    $ brew doctor # (Resolve any any errors or warnings)
+    $ brew install python
+    $ brew install git
+    $ brew install boot2docker
+    $ brew install docker-compose
+    $ brew install pyenv-virtualenvwrapper
+    $ boot2docker init
 
-### Preliminaries
+## Fork and Clone Goldstone Repos
 
-If you're doing this on OS X, you need to install these packages:
+Depending on your contributor status (core or community), you will either create forks of the [goldstone-server](https://github.com/Solinea/goldstone-server) and [goldstone-docker](https://github.com/Solinea/goldstone-docker) Github repositories, or you will be working on on branches from the main repos.
```

Comment:
This offers a choice, either core or community status; then it says what community contributors should do. ("If you are a community contributor...")  Further instructions for core contributors are missing.  

Either fold everything together (which means inserting "your branch or fork" in multiple places) or put in specific instructions for core contributors.